### DISCUSSION TITLE
Get managed cluster resources' details

### DISCUSSION THREAD

**Discussion by geowalrus4gh** [argoproj/argo-cd#14770] [Q&A]
Hi, 

Is there any way to get the resource details from the managed remote cluster? 
For example, get the DNS name from an ingress applied to a remote cluster after application deployment. 
Currently, we can see the resource details from the UI when we click against the resource. 
How to get those details from the admin cluster programmatically? 
Is any managed resource copy available in the admin cluster or 
Can I use argocd cli to retrieve those?

Thanks
George

**Comment by crenshaw-dev** [comment]
You can use `/api/v1/applications/{name}/resource` in the [API](https://argo-cd.readthedocs.io/en/latest/developer-guide/api-docs/) to get the same information you get via the UI. (The UI is just using the API under the hood.)

**Reply by geowalrus4gh** [reply]
yea got it. Thanks. 
BTW since argocd does multi-cluster deployment, how do we compare this with OCM( https://open-cluster-management.io/) ?

**Reply by crenshaw-dev** [reply]
idk much about OCM, but my understanding is that it does not provide any gitops functionality. So both products have their independent purposes.

I believe there has been some effort to automatically translate OCM resources to Argo CD cluster configuration to avoid config duplication. But I've not researched it much.
